content/blog/2023/11/2023-11-06-introducing-2-2-2-java-support-plan.adoc

```diff
@@ -0,0 +1,189 @@
+---
+layout: post
+title: "Introducing the 2 + 2 + 2 Java support plan"
+tags:
+- announcement
+- core
+- developer
+- jenkins
+author: basil
+description: >
+  Jenkins 2.426.1 LTS will support Java 11, 17, and 21.
+  In Fall 2024, Jenkins will require Java 17 or 21.
```

Comment:
```suggestion
  In Fall 2024, Jenkins will require Java 17 or 21, and drop support for Java 11.
```
Suggestion: clearly state that Java 11 won't be supported anymore in Fall 2024.

Reply:
For consistency, I have also updated another occurrence of this phrase in coimmit e589ed023bacec028ea5c5df0c817a069aa67726.